mmap-queue/src/main/java/org/tools4j/mmap/longQueue/api/LongAppender.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 tools4j.org (Marco Terzer, Anton Anufriev)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.tools4j.mmap.longQueue.api;
+
+/**
+ * Long appender.
+ */
+public interface LongAppender extends AutoCloseable {
+    /**
+     * Failed to move to end of the queue
+     */
+    long MOVE_TO_END_ERROR = -1;
+    /**
+     * Failed to wrap region for given position
+     */
+    long WRAP_REGION_ERROR = -2;
+
+    /**
+     * Appends a long value.
+     *
+     * @param value - value, must not be {@link LongQueue#NULL_VALUE}
+     * @return index at which value was appended, negative value if error occurred, see errors in {@link LongAppender}
+     */
+    long append(long value);
+
+    /**
+     * Override to avoid throwing checked exception.
```

Comment:
this should be silent (or no) comment --- javadoc should describe what this method does.

Reply:
addressed in https://github.com/tools4j/mmap/pull/10